src/RedHat.AspNetCore.Server.Kestrel.Transport.Linux/TransportThread.ThreadContext.cs

```diff
@@ -111,6 +112,42 @@ public unsafe ThreadContext(TransportThread transportThread)
                     AioInterop.IoSetup(EventBufferLength, &ctx).ThrowOnError();
                     _aioContext = ctx;
                 }
+
+                // Single reader, single writer queue since all writes happen from the TransportThread and reads happen sequentially
+                // This channel is unbounded which means there's nothing limiting the number of sockets we're accepting.
+                // This is similar to having an unbounded number of thread pool work items queued to invoke a ConnectionHandler
+                // which was the previous pattern, but now it's more explicit. However, it would be good to find a reasonable limit
```

Comment:
> which was the previous pattern

I'm not sure where that happened, previously we were directly invoking `_connectionDispatcher.OnConnection`.

Can you add `(TODO)` at the end of the comment?